Kajabi/sage-lib
# docs(modal): Rails Modal Documentation

## Description

Update Form Modal documentation to better explain props in Rails.

## Screenshots

|  Before  |  After  |
|--------|--------|
|<img width="1588" alt="Screenshot 2023-10-11 at 3 58 24 PM" src="https://github.com/Kajabi/sage-lib/assets/1241836/607466a4-461e-45bf-a93c-588dc87f257b">|<img width="1389" alt="Screenshot 2023-10-11 at 3 58 37 PM" src="https://github.com/Kajabi/sage-lib/assets/1241836/3c10639c-b9cd-4f76-834d-35c26bae2782">|


## Testing in `sage-lib`

- Visit [Modal rails page](http://localhost:4000/pages/component/modal?tab=preview)
- Verify updated docs

## Testing in `kajabi-products`

1. (**N/A**) Modal documentation update.

## Related
